tests/test_pi_extension_install.py

```
@@ -6,15 +6,19 @@
 import fcntl
 import json
 import os
 import signal
 import shutil
+import socketserver
 import subprocess
 import tempfile
+import threading
 import time
 from pathlib import Path
+from typing import Iterator
```

Ruff (0.16.0)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)